tox.ini

```diff
@@ -1,15 +1,16 @@
 [tox]
 isolated_build = true
 skipsdist=true
-envlist = py{38,39},flake8
+envlist = py{310,311,312}
```

Comment:
Similarly to my comment above, is there a default list we can use to avoid manually updating this all the time?